### PR TITLE
Another python solution for day05. Oh, and another one-liner from hell!

### PR DESCRIPTION
**Pull request by Stektpotet**
_Seems the 🐍 way has gained some traction_

one-liner to come...

### CODE REVIEW

File: days/day-05/solutions/one-liner.stektpotet.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import sys; [(print(max(ids)), print(i)) for ids in [{int(''.join('0' if c == 'F' or c == 'L' else '1' for c in k), 2) for k in sys.stdin.read().split('\n')[:-1]}] for i in range(min(ids), max(ids)) if not i in ids]
```
Comment (Arxcis) [inline days/day-05/solutions/one-liner.stektpotet.py:1]: Hmm. Is this really a one-liner though? :eyes: I would argue that `import sys;` is on a separate "line" :P 

Ok it is a one-liner, but it is not a one-statementer. It is a two-statementer :upside_down_face:  Approved!

**Review by Arxcis — APPROVED** [review]
:+1:

File: days/day-05/solutions/day05.stektpotet.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import sys
+ids = {int(''.join('0' if c == 'F' or c == 'L' else '1' for c in k), 2) for k in sys.stdin.read().split('\n')[:-1]}
+print(max(ids))
+for i in range(min(ids), max(ids)):
+    if not i in ids:
+        print(i)
```
Comment (Arxcis) [inline days/day-05/solutions/day05.stektpotet.py:6]: Wow! This is elegant.  Does not require an extra intermediate storage, like many other solutions here. 2x approved! :tada:

**Review by Arxcis — APPROVED** [review]

File: days/day-05/solutions/day05.stektpotet.py (hunk 2 above)
Comment (Stektpotet) [reply days/day-05/solutions/day05.stektpotet.py:6]: ## Now we're thinking with 🐍 
Sidenote:
Turns out I actually did a minor mixup here, though python says **I'll allow it!**,  the preferred way is: `if i not in ids`
Runs just the same either way though 😅

Comment (Arxcis) [reply days/day-05/solutions/day05.stektpotet.py:6]: Wow. Python MVP! :pray:

File: days/day-05/solutions/one-liner.stektpotet.py (hunk 1 above)
Comment (Stektpotet) [reply days/day-05/solutions/one-liner.stektpotet.py:1]: Tell me if you find a way of reading `stdin` wothout using an imported library, and I'll change it to a one-statementer! 🤭

**Pull request merged**